docs/userGuide/syntax/dates.mbdf

```diff
@@ -0,0 +1,105 @@
+## Dates
+
+{% macro njcode(raw) %}<code>{<a/>{ {{ raw }} }}</code>{% endmacro %}
+
+**Markbind supports date formatting and simple calculations** as a Nunjucks filter.
+
+**Syntax:** {{ njcode('baseDate | date(format, daysToAdd)') }}
+
+<span id="main-example">
+<include src="outputBox.md" boilerplate >
+<span id="code">
+
+<box><span>
+{{ njcode('set base1 = "2019-08-12"') }}<br/>
+{{ njcode('set format1 = "DD MM YYYY"') }}<br/>
+{{ njcode('set format2 = "ddd DD/MM"') }}<br/>
+{{ njcode('base1 | date') }} // Mon 12 Aug<br/>
+{{ njcode('base1 | date(format1)') }} // 12 08 2019<br/>
+{{ njcode('base1 | date(format1, 10)') }} // 22 08 2019<br/>
+{{ njcode('base1 | date(format2, 10)') }} // Thu 22/08<br/>
+</span></box>
+
+</span>
+
+<span id="output">
+Mon 12 Aug
+
+12 08 2019
+
+22 08 2019
+
+Thu 22/08
+</span>
+</include>
+
+<box type="info">
+The base date follows the format: `YYYY-MM-DD`
+
+It can be supplied using [page variables](../reusingContents.html#variables) for convenience.
+</box>
+
+
+### Formatting
+
+The date can be formatted to suit your needs
+
+Default format: `ddd D MMM` e.g. Fri 6 Mar
+
+**Brief reference**
+
+Token | Output
+- | -
+D | 1
+Do | 1st
+DD | 01
+M | 1
+MM | 01
+MMM | Jan
+MMMM | January
+YY | 19
+YYYY | 2019
+
+Full formatting reference available [here](https://momentjs.com/docs/#/displaying/format/).
+
+Example
+<span id="examples" class="d-none">
+<include boilerplate src="outputBox.md">
+<span id="code">
```

Comment:
The code example here did not have the box appearance in the final render, unlike the other one.